### PR TITLE
Document for HTMLAnchorElement.target

### PR DESCRIPTION
**Pull request by ShubhamOulkar**
This adds the missing HTMLAnchorElement.target page.

resource link
[target](https://html.spec.whatwg.org/multipage/text-level-semantics.html#htmlanchorelement)

Part of the project to document all interoperable features ([mdn/discussions#476](https://github.com/orgs/mdn/discussions/476))

### CODE REVIEW

File: files/en-us/web/api/htmlanchorelement/target/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+{{ApiRef("HTML DOM")}}
+
+The **`target`** property of the {{domxref("HTMLAnchorElement")}} interface is a string that indicates where to display the linked resource.
+
+It reflects the `target` attribute of the {{HTMLElement("a")}} element.
+
+## Value
+
+A string that indicates where to display the linked resource, or the empty string (`""`) if there is no `target` attribute.
```
Comment (teoli2003) [inline files/en-us/web/api/htmlanchorelement/target/index.md:17]: You need to list the potential value and what they mean:
- `_blank`
- `_self` 
- `_parent`
- `_top`
- a frame name

There is some extra information: https://copyprogramming.com/howto/difference-between-self-top-and-parent-in-the-anchor-tag-target-attribute (Don't copy info from there!). You can add this link to the See Also section, too.

You don't need to go into too much detail (especially about the parts that were frame-specific, as these, though still available, are deprecated).

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/htmlanchorelement/target/index.md (hunk 1 above)
Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlanchorelement/target/index.md:17]: [target values already explain on <a> page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target)

I was thinking of adding some more examples for target values.

Comment (teoli2003) [reply files/en-us/web/api/htmlanchorelement/target/index.md:17]: You still need to explain them here, too. A user doesn't know it has to look there, and it is better to have them on the very same page this is used.

Comment (teoli2003) [reply files/en-us/web/api/htmlanchorelement/target/index.md:17]: An alternative is to list them and link to their explanation elsewhere on MDN.

Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlanchorelement/target/index.md:17]: I created simple example using single <iframe> inside intractive example code editor, which is a <iframe>. 

follow pr [#2712](https://github.com/mdn/interactive-examples/pull/2712)

**Review by teoli2003 — CHANGES_REQUESTED** [review]
(Need to do the same as in https://github.com/mdn/content/pull/31652/files#r1457060901 )

File: files/en-us/web/api/htmlanchorelement/target/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+The **`target`** property of the {{domxref("HTMLAnchorElement")}} interface is a string that indicates where to display the linked resource.
+
+It reflects the [`target`](/en-US/docs/Web/HTML/Element/a#target) attribute of the {{HTMLElement("a")}} element.
+
+## Value
+
+A string representing the target. Its value can be:
+
+- The name of a {{HTMLElement("frame")}}
```
Comment (teoli2003) [inline files/en-us/web/api/htmlanchorelement/target/index.md:19]:
```suggestion
- The name of a {{HTMLElement("frame")}}.
```

File: files/en-us/web/api/htmlanchorelement/target/index.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+```
+
+```js
+const link = document.querySelector(".link1");
+console.log(link.target); // output: "_blank"
+```
+
+Check different values of a target attribute for `<a>` tag.
+
+{{EmbedInteractiveExample("pages/webapi-tabbed/HTMLAnchorElement-target.html", "tabbed-shorter")}}
```
Comment (teoli2003) [inline files/en-us/web/api/htmlanchorelement/target/index.md:35]: Sorry, I didn't notice this earlier.

Interactive examples are not used in this case. We should either use a live sample or nothing.

I think we can use nothing.
```suggestion
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Thank you! Almost ready to land.

File: files/en-us/web/api/htmlanchorelement/target/index.md (hunk 3 above)
Comment (ShubhamOulkar) [reply files/en-us/web/api/htmlanchorelement/target/index.md:35]: OK

**Review by teoli2003 — APPROVED** [review]
👍  Thank you! Very happy to be able to land these PRs!

**Pull request merged**